Extending/Key-Vault/index.md

```diff
@@ -0,0 +1,71 @@
+---
+versionFrom: 9.0.0
+state: complete
+meta.Title: "Azure Key Vault"
+meta.Description: "A guide configuring an Azure Key Vault"
+---
+
+# Configuring an Azure Key Vault
+
+From a security perspective, it's always a good solution to store your application secrets such as a connection string and others keys in an Azure Key Vault. This article tells you how to configure your application so that it's ready to use a Key Vault
+
+## Installing the package
+
+Before you begin, you need to install the `Microsoft.Extensions.Configuration.AzureKeyVault` NuGet package. There are two approaches to installing the package:
+
+1. Use your favorite IDE and open up the NuGet Package Manager to search and install the package
+1. Use the command line to install the package
+
+## Installing through command line
+
+Navigate to your project folder, which is the folder that contains your .csproj file. Now use the following dotnet add package command to install the package:
+
+```
+dotnet add package Microsoft.Extensions.Configuration.AzureKeyVault
+```
+
+## Configuration
+The next step is to add the Azure Key Vault endpoint to the 'appsettings.json' file. 
+
+```json
+{
+  "AzureKeyVaultEndpoint": "https://{your-key-vault-name}.vault.azure.net",
+}
+```
+
+After adding the Key Vault endpoint you have to update the`CreateHostBuilder` method which you can find in the `Program.cs` class. 
```

Comment:
```suggestion
After adding the Key Vault endpoint you have to update the `CreateHostBuilder` method which you can find in the `Program.cs` class. 
```

Reply:
@jonat123 Thanks for your suggestion, just updated 👍